### PR TITLE
"Window-setup-Survey-Native"

### PR DESCRIPTION
**Pull request by MallanagoudaB**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -451,6 +528,19 @@ Before setting up the following Survey application, dependencies given below sho
       git clone -b master https://github.com/ELEVATE-Project/scheduler.git && \
       git clone -b main https://github.com/ELEVATE-Project/observation-survey-projects-pwa.git
       ``` 
+   -  **Windows**
```
Comment (VISHNUDAS-tunerlabs) [inline README.md:531]: @MallanagoudaB Please provide proper alignment

File: README.md [hunk 2]
```diff
@@ -783,6 +956,12 @@ In such cases, you can generate sample user accounts using the steps below. This
     chmod +x insert_sample_data.sh && \
     ./insert_sample_data.sh
     ```
+-   **Windows**
```
Comment (VISHNUDAS-tunerlabs) [inline README.md:959]: @MallanagoudaB adding this here for 12.  **Start The Portal** section. correct the indentation of the section .  make sure that after creating  user account only we ask user to explore the portal

File: README.md [hunk 3]
```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
...
+            ```
+            cd user\src && npx sequelize-cli db:migrate && cd ..\.. &&
+            cd notification\src && npx sequelize-cli db:migrate && cd ..\..
+            ```
 
 7.  **Enabling Citus And Setting Distribution Columns (Optional)**
 
       To boost performance and scalability, users can opt to enable the Citus extension. This transforms PostgreSQL into a distributed database, spreading data across multiple nodes to handle large datasets more efficiently as demand grows.
 
       > NOTE: Currently only available for Linux based operation systems.
```
Comment (Prajwal17Tunerlabs) [inline README.md:698]: @MallanagoudaB You can remove the NOTE from here as you have added windows citus setup as well.

File: README.md [hunk 4]
```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
...
+         cd notification/src && npm install && cd ../.. && \
+         cd interface-service/src && npm install && cd ../.. && \
+         cd scheduler/src && npm install && cd ../.. && \
+         cd observation-survey-projects-pwa && npm install --force && cd ..
+         ```  
+      -  **Windows**
+
+         ```
+         cd samiksha-service & npm install & cd ..\ & ^
+         cd entity-management\src & npm install & cd ..\.. & ^
```
Comment (Prajwal17Tunerlabs) [inline README.md:562]: @MallanagoudaB is entity-management service working for any node version OR only for a specific version. If it works only for a specific node version. Please mention it separately

File: README.md [hunk 5]
```diff
@@ -440,151 +517,230 @@ Before setting up the following Survey application, dependencies given below sho
...
+               ```
+            
+            2. Enable Citus and set distribution columns for `user` database by running the `citus_setup.bat`with the following arguments.
+               ```
+               citus_setup.bat user postgres://postgres:postgres@localhost:9700/users
+               ```
+
 8.  **Insert Initial Data**
 
      - **Ubuntu/Linux/Mac/Windows**
```
Comment (Prajwal17Tunerlabs) [inline README.md:746]: @MallanagoudaB commands mentioned under Step 8 will not work for windows setup. Please check.

File: README.md [hunk 6]
```diff
@@ -783,14 +946,25 @@ In such cases, you can generate sample user accounts using the steps below. This
...
+    ```
 
     After successfully running the script mentioned above, the following user accounts will be created and available for login:
    
-| Email ID                 | Password   | Role                      |
-| ------------------------ | ---------- | ------------------------- |
-| aaravpatel@example.com   | Password1@ | state_educational_officer |
-| arunimareddy@example.com | Password1@ | state_educational_officer |
-| devikasingh@example.com  | Password1@ | state_educational_officer |
+      | Email ID                 | Password   | Role                      |
```
Comment (Prajwal17Tunerlabs) [inline README.md:957]: Check the alignment of this table. It should be more towards left of the documentation.

File: README.md (hunk 5 above)
Comment (MallanagoudaB) [reply README.md:746]: this will work for all OS @Prajwal17Tunerlabs

File: documentation/1.0.0/native/scripts/windows/citus_setup.bat [hunk 7]
```diff
@@ -0,0 +1,77 @@
+@echo off
```
Comment (Prajwal17Tunerlabs) [inline documentation/1.0.0/native/scripts/windows/citus_setup.bat:1]: @MallanagoudaB check if citus is functioning properly in windows native.

**Review by Prajwal17Tunerlabs — APPROVED** [review]
Review completed at 3:10 pm.

**Pull request merged**